docs/actions-new.md

```diff
@@ -52,6 +52,9 @@ For some languages, it may also be necessary to
 **Note:** Steps 3-6 are ripe for automation and should further reduce the touch-points
 for adding a new runtime to the OpenWhisk platform in the future.
 
+[*1]: At this moment 'XXX_THE_END_OF_A_WHISK_ACTIVATION_XXX' is used as the sentinel
```

Comment:
my preference is to inline footnotes into a new section that discusses the protocol.
this could use as a starting point the information @akrabat documented in the wiki and augment it with additional information not present there, testing methodology, ...